CONTRIBUTING.md

```diff
@@ -792,11 +792,25 @@ organization, because of a limitation of GitHub).
 #### Additional notes for pull request reviewers and assignees ####
 
 - NEVER USE GITHUB'S MERGE BUTTON.  Instead, you should either:
-  - run the [`dev/tools/merge-pr.sh`][merge-pr] script (requires
-    having configured gpg with git);
-  - or post a comment containing "@coqbot: merge now" (this is
-    especially convenient for developers who do not have a GPG key and
-    for when you do not have access to a console).
+
+  - post a comment containing "@coqbot: merge now";
+    This is the recommended method and more convenient than the previous
+    script based method (see next bullet) e.g. for developers who do not have
+    a GPG key and for when you do not have access to a console.
+    "coqbot" will **not** check CI status - it is expected that the merger does
+    this manually upfront, but coqbot will deny the merge with an error
+    response in the following cases:
+    - no assignee
```

Comment:
It's more specific than that: the author of the `merge now` comment must be an assignee

Reply:
@Zimmi48 was quick with merging ... shall I do another PR?

Reply:
Right, sorry for merging too fast. Sure, another PR would be welcome.